src/aotcompile.cpp

```diff
@@ -622,7 +704,7 @@ void jl_dump_native_impl(void *native_code,
             emit_result(asm_Archive, asm_Buffer, asm_Name, outputs);
     };
 
-    add_output(*dataM, "unopt.bc", "text.bc", "text.o", "text.s");
+    add_output(*dataM, "unopt.bc", "text.bc", "text.o", "text.s", true);
```

Comment:
I would prefer flipping this so sysimageM contains the crt stubs, so that all of our non-codegen "hacks" end up in the sysimage module and the data module is just raw codegen output.